src/main/java/me/zeroeightsix/kami/module/modules/misc/HighwayTools.kt

```diff
@@ -121,9 +150,10 @@ class HighwayTools : Module() {
     }
 
     private fun doTask(): Boolean {
-        if (!isDone()) {
+        if (!isDone() && !MODULE_MANAGER.getModuleT(LagNotifier::class.java).paused) {
             if (waitTicks == 0) {
                 val blockAction = blockQueue.peek()
+                BaritoneAPI.getProvider().primaryBaritone.pathingControlManager.registerProcess(KamiMod.highwayToolsProcess)
```

Comment:
should this really be registering every time you call doTask? or am I missing something?